src/mono/mono/mini/mini-llvm.c

```diff
@@ -2136,6 +2138,20 @@ set_nonnull_load_flag (LLVMValueRef v)
 	LLVMSetMetadata (v, md_kind, LLVMMDNode (&md_arg, 1));
 }
 
+static void
+set_nontemporal_flag (LLVMValueRef v)
+{
+	LLVMValueRef md_arg;
+	int md_kind;
+	const char *flag_name;
+
+	// FIXME: Cache this
+	flag_name = "nontemporal";
+	md_kind = LLVMGetMDKindID (flag_name, strlen (flag_name));
+	md_arg = LLVMMDString ("<index>", strlen ("<index>"));
```

Comment:
um, what is `<index>` ? clang emits ` = !{i32 1}`, see https://godbolt.org/z/aEmvGo
And this code emits:
```llvm
  ...
  store <4 x float> %6, i64 %t18, align 16, !nontemporal !0
  ret void
}

!0 = !{!"<index>"}
```

Reply:
The manual says:
The optional !nontemporal metadata must reference a single metadata name <index> corresponding to a metadata node with one i32 entry of value 1.  Not sure how to generate that, but it doesn't seem to matter.